readme.md

```diff
@@ -16,7 +16,24 @@ the square brackets (for multiple-choice questions).
 Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back. (Assume the server responds with a JSON array of objects representing gifs.)
 
 Your Answer:
-```js
+```
+$(document).ready(function() {
+  $("h1").on("click", function() {
+  var url = "http://kittengifs.com/gifs"
+  $.ajax({
+    url: url,
+    type: "get",
+    datatype: "json"
+    })
+    .done(function(){
+      console.log("Success!")
+  }).fail(function(){
```

Comment:
really close! This will make a sucessful ajax call, but the it will always just log out `"Success!"`. If we send `response` in as an argument we can see what it brings back by going the following:

```js
.done(function(response){
  console.log(response)
}
```

Then you could grab the length of the response instead.